datalab/mlalpha/_cloud_models.py

```diff
@@ -51,7 +43,9 @@ def _retrieve_models(self, page_token, page_size):
     page_token = list_info.get('nextPageToken', None)
     return models, page_token
 
-  def __iter__(self):
+  def get_iterator(self):
```

Comment:
why the name change to get_iterator()? Also do the same change to the Jobs class.